packages/@o3r/rules-engine/src/engine/operator/operator.interface.ts

```diff
@@ -1,3 +1,5 @@
+import type { Facts } from '../engine.interface';
```

Comment:
This is actually quite close to be a circular dependency (on the file path, not the imported interface).
May be we should take care of it